ConsoleApp1/Answer.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
```

Comment:
Create VoteSystem where you will place all logic with voting, display, remove, add, answer
Create new entity VoteOption, pass VoteOption to Answer's constructor